solr/core/src/java/org/apache/solr/cloud/api/collections/SplitShardCmd.java

```diff
@@ -80,6 +80,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * Index split request processed by Overseer. Requests from here go to the host of the parent shard,
```

Comment:
Not necessarily by the Overseer in Solr 9?  Probably not a good idea to say who one's caller is in general as it can get stale and may not matter.